### ISSUE TITLE
Can't vote after guest added song

### ISSUE THREAD

**Issue by ahelfins** [ahelfins/Democratic-DJ#11]

**Comment by jinyoung-lim** [comment]
Not being able to vote on multiple songs would be a bigger issue (comment under Issue #12 ). We should make sure that the votes are being updated on firebase too.

**Comment by jinyoung-lim** [comment]
It seems like session data's this.songVotes is not working? When trying to vote (as a guest)  it says that the song has "-1" votes whether it has been upvoted or downvoted. Then the votes do not get updated in firebase. I observed that one song's downvote in guest song list page incremented but was not updated in firebase. As a result, when another song was added, the downvote went from 1 to 0 again.

 I think if we have to choose between 1) being able to vote (and update the votes in firebase) but cannot regulate multiple votes on the same song and 2) not being able to vote but can regulate overlapping votes, we should choose the former.
